### ISSUE TITLE
Create a release to prevent `version conflict(s) in library dependencies`

### ISSUE THREAD

**Issue by froth** [pureconfig/pureconfig#1609]
Hi,

currently one has to suppress the following error when using `pureconfig-http4s` with a more current `http4s` version:

```
[error] (dependencyTreeIgnoreMissingUpdate) found version conflict(s) in library dependencies; some are suspected to be binary incompatible:
[error]                                                                                        
[error]         * org.typelevel:cats-parse_3:1.0.0 (early-semver) is selected over 0.3.9
[error]             +- org.http4s:http4s-core_3:0.23.25 ()                (depends on 1.0.0)  
[error]             +- org.http4s:http4s-core_3:0.23.18 ()                (depends on 0.3.9)   
[error] 
[error]                                                                                        
[error] this can be overridden using libraryDependencySchemes or evictionErrorLevel
[error] Total time: 6 s, completed Jan 18, 2024, 9:27:03 AM
```
The update to org.http4s:http4s-core_3:0.23.25 has already been merged to this repo: https://github.com/pureconfig/pureconfig/pull/1604

So a release of `pureconfig-http4s` would fix these problems.

**Comment by ruippeixotog** [comment]
Hi @froth, thanks for letting me know. I have just published v0.17.5.

**Comment by froth** [comment]
Thank you! Works without the suppress now :)
